yellowbrick/features/pca.py

```diff
@@ -200,6 +254,7 @@ def fit(self, X, y=None, **kwargs):
         self : visualizer
             Returns self for use in Pipelines
         """
+        # Call super fit to compute features, classes, colors, etc.
         super(PCADecomposition, self).fit(X=X, y=y, **kwargs)
         self.pca_transformer.fit(X)
         self.pca_components_ = self.pca_transformer.named_steps["pca"].components_
```

Comment:
Would you mind adding `pca_components_` and any other learned parameters (e.g. those with an underscore suffix that are added in fit) to the docstring please?

Reply:
Sure!